Azure/sonic-buildimage
# [libteam]: Add Fast-Reboot mode for teamd

**- What I did**
Implemented stop mode for fast-reboot. When you send SIGUSR2 to teamd, it will stop in FR mode. That means that:
- teamd will send last lacp packet
- teamd will not remove PortChannels
We have very similar mode: WR, but WR mode activated by SIGUSR1. Also it dumps LACP state to the disk.

**- How I did it**
Catching USR2 signal.

**- How to verify it**
Build changed teamd. Install and run it. Capture packets and send teamd USR2 signal. You will see that teamd left all PortChannels untouched, sends LACP packets right before the stop.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
